CHANGELOG.md

```diff
@@ -444,15 +444,18 @@
 - Show error message when the user tries to enter an invalid date on the `dates` step. Covers `planned_start_date`, `planned_end_date`, `actual_start_date`, `actual_end_date`
 - Catch encoding errors when uploading Transactions with invalid characters
 
-## [unreleased]
+## [release-27] - 2020-12-21
```

Comment:
I think the date needs to be amended. Other than that it looks fine so I'll approve

Reply:
Whoops!